### PR TITLE
Add code coverage and GitHub actions badges to the README

### PR DESCRIPTION
**Pull request by wilfwilson**
This adds a badge to the README showing the code coverage percentage and the build status in GitHub actions, both for the master branch.

Since we merge our stable branches into our master branch periodically, I don't see how we can easily have the badge showing the status for the master branch in the master branch, but showing the status for the stable branch in the stable branch. So I've just gone for the master branch everywhere.

You can see the result of this by scrolling down to [the README on this page](https://github.com/wilfwilson/Digraphs/tree/add-more-badges-to-README).

The GitHub actions badge will show 'no status' until we merge stable-1.3 into master. This is because the master branch doesn't yet have the fully up-to-date CI.

### CODE REVIEW

**Review by james-d-mitchell — APPROVED** [review]

**Comment by james-d-mitchell** [comment]
@wilfwilson  thanks for this and the other PRs! Is there any reason that we can't merge the CI changes in the stable branch into master?

**Comment by wilfwilson** [comment]
No reason, we should definitely merge `stable-1.3` into `master` once this PR is merged!

**Pull request merged**
